packages/badge/src/css/index.js

```diff
@@ -11,82 +19,87 @@ export default {
-    borderColor: core.colors.green
+    color: colorsTextIcon.highOnDark,
+    backgroundColor: colorsGreen.base,
+    borderColor: colorsGreen.base
   },
   [defaultWithColor(colors.yellow)]: {
-    color: core.colors.black,
-    backgroundColor: core.colors.yellow,
-    borderColor: core.colors.yellow
+    color: '#181818',
+    backgroundColor: colorsYellow.base,
+    borderColor: colorsYellow.base
   },
   [defaultWithColor(colors.red)]: {
-    color: core.colors.white,
-    backgroundColor: core.colors.red,
-    borderColor: core.colors.red
+    color: colorsTextIcon.highOnDark,
+    backgroundColor: colorsRed.base,
+    borderColor: colorsRed.base
   },
   [defaultWithColor(colors.blue)]: {
-    color: core.colors.white,
-    backgroundColor: core.colors.blue,
-    borderColor: core.colors.blue
+    color: colorsTextIcon.highOnDark,
+    backgroundColor: colorsBlue.base,
+    borderColor: colorsBlue.base
   },
 
   [subtleThemeWithColor(themeNames.dark, colors.gray)]: {
-    color: core.colors.gray02,
-    borderColor: core.colors.gray02
+    color: colorsTextIcon.lowOnDark,
+    borderColor: colorsBorder.highOnDark
   },
 
   [subtleThemeWithColor(themeNames.dark, colors.green)]: {
-    color: core.colors.green,
-    borderColor: core.colors.green
+    color: colorsGreen.base,
+    borderColor: colorsGreen.base
   },
 
   [subtleThemeWithColor(themeNames.dark, colors.yellow)]: {
-    color: core.colors.yellow,
-    borderColor: core.colors.yellow
+    color: colorsYellow.base,
+    borderColor: colorsYellow.base
   },
 
   [subtleThemeWithColor(themeNames.dark, colors.red)]: {
-    color: core.colors.red,
-    borderColor: core.colors.red
+    color: colorsRed.base,
+    borderColor: colorsRed.base
   },
 
   [subtleThemeWithColor(themeNames.dark, colors.blue)]: {
-    color: core.colors.blue,
-    borderColor: core.colors.blue
+    color: colorsBlue.base,
+    borderColor: colorsBlue.base
   },
 
   [subtleThemeWithColor(themeNames.light, colors.gray)]: {
-    color: core.colors.gray04,
-    backgroundColor: transparentize(0.85, core.colors.gray03)
+    color: colorsTextIcon.highOnLight,
+    backgroundColor: colorsBackgroundUtility[25],
+    borderColor: colorsBackgroundUtility[25]
   },
   [subtleThemeWithColor(themeNames.light, colors.green)]: {
-    color: core.colors.gray04,
-    backgroundColor: transparentize(0.85, core.colors.green)
+    color: colorsTextIcon.highOnLight,
+    backgroundColor: colorsGreen[1],
+    borderColor: colorsGreen[1]
   },
   [subtleThemeWithColor(themeNames.light, colors.yellow)]: {
-    color: core.colors.gray04,
-    backgroundColor: transparentize(0.85, core.colors.yellow)
+    color: colorsTextIcon.highOnLight,
+    backgroundColor: colorsYellow[1],
+    borderColor: colorsYellow[1]
   },
   [subtleThemeWithColor(themeNames.light, colors.red)]: {
-    color: core.colors.gray04,
-    backgroundColor: transparentize(0.85, core.colors.red)
+    color: colorsTextIcon.highOnDark,
```

Comment:
After seeing more comments and becoming more careful, I do see something that looks odd to my eye, and checking storybook does not seem quite right:

That is, red subtle on Light theme.

<img width="156" alt="Screen Shot 2020-01-27 at 4 06 06 PM" src="https://user-images.githubusercontent.com/552044/73221785-f7c61680-411e-11ea-9aca-d998a742bb60.png">

Reply:
Think this should be highOnLight.